src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.cpp

```diff
@@ -148,6 +154,11 @@ namespace winrt::PowerToys::PowerAccentKeyboardService::implementation
 
     bool KeyboardListener::OnKeyUp(KBDLLHOOKSTRUCT info) noexcept
     {
+        if (m_shiftPressed && (info.vkCode == VK_LSHIFT || info.vkCode == VK_RSHIFT))
```

Comment:
Let's consider the next scenario:
1. <kbd>Left Shift</kbd> is down
2. <kbd>Right Shift</kbd> is down
3. <kbd>Right Shift</kbd> is up

Then `m_shiftPressed` will be set to false, even though <kbd>Left Shift</kbd> is still pressed.